scripts/Jenkinsfile-deploy

```diff
@@ -97,45 +91,59 @@ node('WWW' && '!master') {
         return
     }
 
+
+    // We copy everything to a staging directory to avoid 404 while the job is running
+    String stagingDirectory = env.SWF_STAGING_DIR?: "/tmp/software-factory-staging/"
```

Comment:
How do you mean with macro?

Reply:
e.g. 
SWF_STAGING_PATH = "/tmp/software-factory-staging/" on top of the file (not sure how the syntax will be exactly though Like in C  #deinfe MACRO_NAME Value) 

Then assigning SWF_STAGING_PATH to stagingDirectory. This is what I meant by my comment.

Reply:
Sure, but the `stagingDirectory` is only used in the scope underneath its definition, so not sure I see the value in defining it up top right now.

Reply:
Alright! If those paths are local to stage "Setup staging", Just thinking if it would be better to put those string assignment statements inside the state?